docs/dockerhub.md

```diff
@@ -0,0 +1,106 @@
+# jpo-ode
+
+## GitHub Repository Link
+https://github.com/usdot-jpo-ode/jpo-ode
+
+## Purpose
+The purpose of the Operational Data Environment is to ingest, process and disseminate connected vehicle data in the form of messages defined in J2735. The primary program utilizes a number of internal modules and external submodules.
+
+## How to pull the latest image
+The latest image can be pulled using the following command:
+> docker pull usdotjpoode/jpo-ode:latest
+
+## Required environment variables
+The image expects the following environment variables to be set:
+- DOCKER_HOST_IP
+- DOCKER_SHARED_VOLUME
+
+## Direct Dependencies
+The ODE will fail to start up if the following containers are not already present:
+- Kafka
```

Comment:
We can configure this to use confluent rather than a kafka container right? Maybe reword this to say:
```
dependencies
- Kafka or Confluent and related requirements
```